shared/src/business/entities/Practitioner.ts

```diff
@@ -199,12 +199,15 @@ export class Practitioner extends User {
     return Practitioner.VALIDATION_RULES as any;
   }
 
-  toRawObject() {
+  toRawObject(options: { removeValidationProperties?: boolean } = {}) {
+    const { removeValidationProperties = true } = options;
     const result = super.toRawObject() as any;
 
     // We don't want to persist these values as they are only used for validation
-    result.confirmEmail = undefined;
-    result.updatedEmail = undefined;
+    if (removeValidationProperties) {
+      result.confirmEmail = undefined;
+      result.updatedEmail = undefined;
+    }
 
     return result;
   }
```

Comment:
Thought about this for a while, and really the idea of tying `toRawObject` to database code is silly and outdated. I like this without the boolean trap

Reply:
Especially since we have mapper functions to deal with this type of problem now